identity-apps-core/apps/authentication-portal/src/main/webapp/templates/internalWait.jsp

```diff
@@ -0,0 +1,290 @@
+<%--
+ ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ ~
+ ~ This software is the property of WSO2 LLC. and its suppliers, if any.
+ ~ Dissemination of any information or reproduction of any material contained
+ ~ herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+ ~ You may not alter or remove any copyright or other notice from copies of this content.
+--%>
+
+<%--  
```

Comment:
JSP apps are not minified during build, therefore the end user will receive the comments as well, causing downloading extra kb. Shall we move this to an issue comment and put the link to it here?

Reply:
Addressed